var/Widget/Options/Discussion.php

```diff
@@ -127,7 +127,7 @@ public function form(): Form
             $this->options->commentDateFormat,
             _t('评论日期格式'),
             _t('这是一个默认的格式,当你在模板中调用显示评论日期方法时, 如果没有指定日期格式, 将按照此格式输出.') . '<br />'
-            . _t('具体写法请参考 <a href="http://www.php.net/manual/zh/function.date.php">PHP 日期格式写法</a>.')
+            . _t('具体写法请参考 <a href="https://www.php.net/manual/zh/function.date.php">PHP 日期格式写法</a>.')
```

Comment:
```suggestion
            . _t('具体写法请参考 <a href="https://www.php.net/date">PHP 日期格式写法</a>.')
```

Reply:
What is this for?

Reply:
Let the php document choose its own language instead of forcing the Chinese document.

Reply:
I think it's out of scope for this PR, but I'll let the maintainers decide and have enabled "allow edits by maintainers".